Search_object and shortname

Does the `search_object` method expect a shortname column? 

```
t = db.search_object(
                simbad_names, output_table="Sources", fmt="astropy", fuzzy_search=False
            )

tests/scheduled_checks.py:48: 
_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ 
../../../../miniforge3/envs/template-dev/lib/python3.13/site-packages/astrodbkit/utils.py:26: in wrapper
    return f(*args, **kwargs)
../../../../miniforge3/envs/template-dev/lib/python3.13/site-packages/astrodbkit/astrodb.py:570: in search_object
    filters = [self.metadata.tables[k].columns[v].ilike(f"{n}") for n in name]
_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ 

self = <sqlalchemy.sql.base.ReadOnlyColumnCollection object at 0x141491940>, key = 'shortname'

    def __getitem__(
        self, key: Union[str, int, slice, Tuple[Union[str, int], ...]]
    ) -> Union[ReadOnlyColumnCollection[_COLKEY, _COL_co], _COL_co]:
        try:
            if isinstance(key, (tuple, slice)):
                if isinstance(key, slice):
                    cols = (
                        (sub_key, col)
                        for (sub_key, col, _) in self._collection[key]
                    )
                else:
                    cols = (self._index[sub_key] for sub_key in key)
    
                return ColumnCollection(cols).as_readonly()
            else:
>               return self._index[key][1]
E               KeyError: 'shortname'

../../../../miniforge3/envs/template-dev/lib/python3.13/site-packages/sqlalchemy/sql/base.py:1605: KeyError
```

--- a/astrodbkit/astrodb.py
+++ b/astrodbkit/astrodb.py
@@ -497,7 +497,7 @@ class Database:
         name,
         output_table=None,
         resolve_simbad=False,
-        table_names={"Sources": ["source", "shortname"], "Names": ["other_name"]},
+        table_names={"Sources": ["source"], "Names": ["other_name"]},
         fmt="table",
         fuzzy_search=True,
         verbose=True,
@@ -517,7 +517,7 @@ class Database:
             Get additional names from Simbad. Default: False
         table_names : dict
             Dictionary of tables to search for name information. Should be of the form table name: column name list.
-            Default: {'Sources': ['source', 'shortname'], 'Names': 'other_name'}
+            Default: {'Sources': ['source'], 'Names': 'other_name'}
         fmt : str
             Format to return results in (pandas, astropy/table, default). Default is astropy table
         fuzzy_search : bool
@@ -553,6 +553,7 @@ class Database:
 
         # Verify provided tables exist in database
         for k in table_names.keys():
+            print(f"Using table '{k}' with columns {table_names[k]} for matching object names")
             if k not in self.metadata.tables:
                 raise RuntimeError(f"Table {k} is not in the database")
 

--- a/astrodbkit/astrodb.py
+++ b/astrodbkit/astrodb.py
@@ -553,7 +553,8 @@ class Database:
 
         # Verify provided tables exist in database
         for k in table_names.keys():
-            print(f"Using table '{k}' with columns {table_names[k]} for matching object names")
+            if verbose:
+                print(f"Using table '{k}' with columns {table_names[k]} for matching object names")
             if k not in self.metadata.tables:
                 raise RuntimeError(f"Table {k} is not in the database")
 

